src/languages/java.js

```diff
@@ -31,7 +31,7 @@ export default function (hljs) {
     variants: [
       { begin: '\\b(0[bB][01]+([01_]*[01]+)?)[lL]?' }, // binary
       { begin: '\\b(0[0-7]+([0-7_]*[0-7]+)?)[dDfFlL]?' }, // octal
-      { begin: '\\b[\\d]+([\\d_]*[\\d]+)?[lL]?' }, // decimal
+      { begin: '\\b[\\d]+[\\d_]*[\\d]+[lL]?' }, // decimal
```

Comment:
Doesn't this require all decimals be 2 digits long now?

Reply:
It was only a test for me, will add this again now

Reply:
You don't have to push your tests. :-)

Reply:
I use Travis at the moment to test, because I haven't the repository checked out on this machine. Developing online with Github. Strange I know...

Reply:
Oh yeah, that's going to be MUCH harder...